src/connector.ts

```diff
@@ -94,9 +102,11 @@ async function setUpCljsRepl(session, build) {
     namespace.updateREPLSessionType();
 }
 
-function getFigwheelMainBuilds() {
-    let res = fs.readdirSync(state.getProjectRoot());
-    let builds = res.filter(x => x.match(/\.cljs\.edn/)).map(x => x.replace(/\.cljs\.edn$/, ""));
+async function getFigwheelMainBuilds() {
+    let res = await vscode.workspace.fs.readDirectory(state.getProjectRootUri())
+    let builds = res
+        .filter(([name, type]) => type != vscode.FileType.Directory && name.match(/\.cljs\.edn/))
```

Comment:
This is a fairly minor thing, but it's good to use `!==` instead of `!=`. The `!=` and `==` are used elsewhere in the code in some places, but I like to keep new code using `!==` and `===`. This follows and airbnb style (which we want to follow with Calva as it's a pretty standard guide) here: https://github.com/airbnb/javascript#comparison-operators--equality, and reasoning is here: https://eslint.org/docs/rules/eqeqeq.html